tests/network-setups/fabric/dev/Makefile

```diff
@@ -100,7 +99,8 @@ deploy-cc-local: setup-cc-local
 
 
 .fabric-setup:
-	curl -sSL https://bit.ly/2ysbOFE | bash -s -- $(FABRIC_VERSION) $(FABRIC_CA_VERSION) $(FABRIC_COUCH_VERSION) -s
+	curl -sSLO https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh && chmod +x install-fabric.sh
+	./install-fabric.sh --fabric-version $(FABRIC_VERSION) --ca-version $(FABRIC_CA_VERSION) docker binary
```

Comment:
Looks like the `install-fabric.sh` script needs to be checked in, no?

Reply:
It is downloaded in line 102. These are the fabric steps to download images and binaries, don't want to put script created by Fabric people in codebase, its better to download everytime I think.